src/main/kotlin/racing/domain/CarRacing.kt

```diff
@@ -0,0 +1,28 @@
+package racing.domain
+
+class CarRacing {
+    private val recorder = CarRacingRecorder
+    fun race(carNames: List<String>, tryCount: Int): CarRacingResult {
```

Comment:
```suggestion
    private val recorder = CarRacingRecorder
    
    fun race(carNames: List<String>, tryCount: Int): CarRacingResult {
```
 프로퍼티와 fun 사이에는 공백이 있으면 좋을 것 같네요.

Reply:
저도 이런 부분 신경 쓰는 편인데, 최근 intellij 업데이트부터 이런식으로 커밋 author 가 프로퍼티나 메서드 위에 노출되어 상대적으로 더 놓치게 되는 것 같아요.
<img width="532" alt="스크린샷 2023-11-02 오후 11 00 03" src="https://github.com/next-step/kotlin-racingcar/assets/32588087/48637033-5cc5-4421-b851-b19c7bb99364">